controller/workitem_blackbox_test.go

```diff
@@ -198,7 +198,7 @@ func (s *WorkItemSuite) TestPagingDefaultAndMaxSize() {
 	limit = 1000
 	_, result = test.ListWorkitemsOK(s.T(), context.Background(), nil, s.workitemsCtrl, space.SystemSpace, nil, nil, nil, nil, nil, nil, nil, nil, &limit, &offset, nil, nil, nil)
 	// then
-	if !strings.Contains(*result.Links.First, "page[limit]=100") {
+	if !strings.Contains(*result.Links.First, "page[limit]=500") {
 		assert.Fail(s.T(), "Limit is more than max", "Expected limit to be %d, got %v", 100, *result.Links.First)
```

Comment:
Shouldn't the 100 in this line be 500?  Since you went to the trouble of creating pageSizeMax constant why not use it here and in the string check on the previous line so this code doesn't have to change again when our page size changes to something else?

Reply:
What happens in 2 weeks when we have more than 500 work items?

Reply:
@stevengutz As mentioned in the description this is a temporary fix in backend. The actual fix would lie in the front-end. The UI does not support pagination as of now. Loading all the data from db at once would lead to performance issues in planner. This PR would be reverted once we have proper pagination in the UI.

Reply:
@stevengutz thanks for pointing out. The `100` should actually be `1000` because that is the expected limit as mentioned in line number 198

Reply:
@stevengutz I think this is what you're looking for: https://github.com/DhritiShikhar/almighty-core/pull/3